### ISSUE TITLE
Add Python dependency to AUGUSTUS?

### ISSUE THREAD

**Issue by krometis** [easybuilders/easybuild-easyconfigs#12660]
I’m trying to build `AUGUSTUS-3.4.0-foss-2020b.eb` and it's failing with the following error:
```
cd ../..; tests/auxprogs/homGeneMapping/test_homGeneMapping.py --clean
/usr/bin/env: python3: No such file or directory
make[2]: *** [clean] Error 127
```
I think this is because the first line of the test wants python3
```
$ head -1 /dev/shm/jkrometi/build/AUGUSTUS/3.4.0/foss-2020b/Augustus-3.4.0/tests/auxprogs/homGeneMapping/test_homGeneMapping.py 
#!/usr/bin/env python3
```
and we don’t have system python3:
```
$ python3 --version
bash: python3: command not found...
Similar command is: 'python'
$ python --version
Python 2.7.5
```
So I think we might need to add python3 (I assume `Python/3.8.6-GCCcore-10.2.0`) to the list of dependencies for `AUGUSTUS-3.4.0-foss-2020b.eb`. I tested this on my machine and it fixed the problem. Hopefully that makes sense and I'm not missing anything...

**Comment by Micket** [comment]
Yes that sounds reasonable. Would you like to open a PR adding `('Python', '3.8.6'),` to dependencies?

**Comment by krometis** [comment]
@Micket Okay I hadn't done that before but I think I figured it out.
https://tenor.com/view/ralphwiggum-smart-simpsons-gif-5270199

**Comment by boegel** [comment]
This was fixed for `foss/2020b` in #12662, but do we also want to fix this for older toolchains?

If so, we'll need to rename those `AUGUSTUS` easyconfigs to have a `-Python-*` versionsuffix (unless we include both `Python` 2.x and 3.x via `multi_deps`, assuming that AUGUSTUS is compatible with both...)

**Comment by branfosj** [comment]
`Python` is a dependency in all current `AUGUSTUS` easyconfigs, so closing this issue as complete.
